### PR TITLE
[JS] Topic: Operators in Javascript

### PR DESCRIPTION
**Pull request by rwiteshbera**
<hr>

## Description 📜
Fixed Issue No. #202

Directory
```
Topics/5.Syntax/2.Operators
```

Discussed various Javascript Operators
1. Assignment Operator
2. Arithmetic Operator
3. Comparison Operator
4. Bitwise Operator
5. Logical Operator
6. Ternary Operator

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] JavaScript

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by paramsiddharth** [comment]
@Creatoon @dakshsethi @s4-2 Please do a speedy review. This is a recreation of #536 to get rid of the conflicts.

**Review by paramsiddharth — APPROVED** [review]

**Review by s4-2 — APPROVED** [review]
Looks good to me.

File: JavaScript/Topics/5.Syntax/2.Operators/1.Assignment_Operators/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# Assignment Operators
+[Back to Operators in Javascript](../README.md#operators-in-javascript)
+
+
+#### 1. Values Assignment (=)
+```js
+let count = 5;
+```
+In this example we use (=) assignment operator to assign the value 5 (Right Operand) to the counter variable (Left Operand).
```
Comment (Creatoon) [inline JavaScript/Topics/5.Syntax/2.Operators/1.Assignment_Operators/README.md:9]:
```suggestion
In this example we use (=) assignment operator to assign the value 5 (Right Operand) to the count variable (Left Operand).
```

File: JavaScript/Topics/5.Syntax/2.Operators/1.Assignment_Operators/README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+Or,
+```js
+let count = count ** x;
+```
+
+Example
+```js
+let count = 5;
+count **= 2; // 5 to the power 2
+console.log(count); // 25
```
Comment (Creatoon) [inline JavaScript/Topics/5.Syntax/2.Operators/1.Assignment_Operators/README.md:75]:
```suggestion
console.log(count); // Output: 25
```

File: JavaScript/Topics/5.Syntax/2.Operators/2.Arithmetic_Operators/README.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+```js
+console.log(5*8);
+// expected output: 40
+```
+
+#### 4. Division (/)
+The division operator (/) is used to divide one variable by another.
+```js
+console.log(8/3);
+// expected output: 2
```
Comment (Creatoon) [inline JavaScript/Topics/5.Syntax/2.Operators/2.Arithmetic_Operators/README.md:36]:
```suggestion
// expected output: 2.6666666666666665
```
I am pointing calculation to this variable only, correct in others if they are incorrect

File: JavaScript/Topics/5.Syntax/2.Operators/2.Arithmetic_Operators/README.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+// expected output: -0
+```
+
+```js
+console.log(5%1);
+// expected output: 0
+```
+
+#### 6. Increment (++)
+The increment operator (++) is used to increment a variable by one.
```
Comment (Creatoon) [inline JavaScript/Topics/5.Syntax/2.Operators/2.Arithmetic_Operators/README.md:60]:
```suggestion
The increment operator (++) is used to increment a number variable by one unit.
```

Comment (Creatoon) [reply JavaScript/Topics/5.Syntax/2.Operators/2.Arithmetic_Operators/README.md:60]: Also correct at other places

File: JavaScript/Topics/5.Syntax/2.Operators/3.Comparison_Operators/README.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+Comparison operators are used to compare values and returns boolean value (true or false).
+
+#### 1. Equal To Operator (==)
+It will return true if both values are equal, otherwise false. It does not check data type of values
+but only checks if they are equal.
+
+```js
+console.log(5==5); // true
+console.log(5=='5'); // false
```
Comment (Creatoon) [inline JavaScript/Topics/5.Syntax/2.Operators/3.Comparison_Operators/README.md:13]:
```suggestion
console.log(5=='5'); // true
```

**Review by Creatoon — CHANGES_REQUESTED** [review]

**Comment by Creatoon** [comment]
@rwiteshbera Check for other mistakes also, if present then fix them, i will approve

File: JavaScript/Topics/5.Syntax/2.Operators/2.Arithmetic_Operators/README.md (hunk 3 above)
Comment (rwiteshbera) [reply JavaScript/Topics/5.Syntax/2.Operators/2.Arithmetic_Operators/README.md:36]: Sir, I am getting 2 here. I have also checked rest all the other calculations.
Screenshot from 2021-09-22 09-40-47

**Comment by rwiteshbera** [comment]
@Creatoon Sir, I have fixed all the issues that you have mentioned, Kindly check it once.

**Review by Creatoon — APPROVED** [review]

**Comment by paramsiddharth** [comment]
image

**Pull request closed without merging**